kategory/src/main/kotlin/kategory/instances/IdInstances.kt

```diff
@@ -41,3 +41,14 @@ interface IdInstances :
             fa.ev().value
 
 }
+
+object IdSemigroupK : SemigroupK<Id.F>, GlobalInstance<SemigroupK<Id.F>>() {
+    override fun <A> combineK(x: HK<Id.F, A>, y: HK<Id.F, A>): HK<Id.F, A> = y
+}
+
+class IdMonoidK(val a: Any) : MonoidK<Id.F>, GlobalInstance<MonoidK<Id.F>>() {
```

Comment:
I have removed both instances and started using OptionInstances and OptionTInstances to test the laws, since both support monoid and semigroup. It's pushed.